nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java

```diff
@@ -163,10 +164,13 @@ protected File getStudyPolicyXML()
     @LogMethod
     protected void populateInitialData() throws Exception
     {
-        List<ModulePropertyValue> props = new ArrayList<>();
-        props.add(new ModulePropertyValue("EHR", "/" + getProjectName(), "EHRCustomModule", "NIRC_EHR"));
         goToProjectHome();
-        setModuleProperties(props);
+        List<ModuleProperty> props = List.of(
+                new ModuleProperty("EHR", "/" + getProjectName(), "EHRCustomModule", "NIRC_EHR"),
+                new ModuleProperty("NIRC_EHR", "/", "NIRCOrchardFileLocation", orchardFileLocation)
+        );
```

Comment:
There is a  hidden dependency on `setOrchardFileLocation` here. It would be more clear to create the directory here.
```suggestion
        FileUtils.deleteQuietly(orchardFileLocation);
        FileUtils.forceMkdir(orchardFileLocation);
        List<ModuleProperty> props = List.of(
                new ModuleProperty("EHR", "/" + getProjectName(), "EHRCustomModule", "NIRC_EHR"),
                new ModuleProperty("NIRC_EHR", "/", "NIRCOrchardFileLocation", orchardFileLocation)
        );
```
Should also remove `setOrchardFileLocation`.